inst/app.crunch.io/api/datasets/1/folders/1.json

```diff
@@ -0,0 +1,35 @@
+{
+    "self": "https://app.crunch.io/api/datasets/1/folders/1/",
+    "element": "shoji:catalog",
+    "size": 5,
+    "body": {
+        "name": "Group 1"
+    },
+    "index": {
+        "../3/": {
+            "type": "folder",
+            "name": "Nested"
+        },
+        "../../variables/birthyr/": {
```

Comment:
the last path segment here should be the variable id, not the alias

Reply:
yes, though these fixtures have arbitrarily used informative ids, just for test readability. The "id" member of the tuple should be updated to match the URL segment, but I'll take that separately.